cmake/external_mlir.cmake

```diff
@@ -24,7 +24,13 @@ set(MLIR_LLVM_COMMIT_ID 75990950)
 set(MLIR_COMMIT_ID 5e64e536)
 
 # MLIR environment variables. Some of them are used by LIT tool.
-set(MLIR_PROJECT_ROOT ${CMAKE_CURRENT_BINARY_DIR}/mlir_project)
+
+if (NGRAPH_USE_PREBUILT_MLIR)
+    set(MLIR_PROJECT_ROOT ${MLIR_LLVM_PREBUILT_PATH}/mlir_project)
```

Comment:
is `mlir_project` necessary? That's a nGraph directory. We should be able to use this flag to compile nGraph with a vanilla llvm/mlir repo. Similar to what PlaidML is doing.